doc/init.md

```diff
@@ -53,11 +53,12 @@ Paths
 
 All three configurations assume several paths that might need to be adjusted.
 
-Binary:              `/usr/bin/bitcoind`
-Configuration file:  `/etc/bitcoin/bitcoin.conf`
-Data directory:      `/var/lib/bitcoind`
-PID file:            `/var/run/bitcoind/bitcoind.pid` (OpenRC and Upstart) or `/run/bitcoind/bitcoind.pid` (systemd)
-Lock file:           `/var/lock/subsys/bitcoind` (CentOS)
+	Binary:              /usr/bin/bitcoind
+	Configuration file:  /etc/bitcoin/bitcoin.conf
+	Data directory:      /var/lib/bitcoind
+	PID file:            /var/run/bitcoind/bitcoind.pid (OpenRC and Upstart) or
+                         /run/bitcoind/bitcoind.pip (systemd)
```

Comment:
What is the point of mixing tabs and spaces? Why is the file extension changed to pip?

Reply:
The file extension change was a typo. Fixed now.

Mixing tabs and spaces is what https://github.com/bitcoin/bitcoin/blob/master/doc/tor.md#L24-L27 does. Do you think we should change it everywhere?